akd_mysql/src/mysql.rs

```diff
@@ -794,7 +795,10 @@ impl Storage for AsyncMySqlDatabase {
 
         match self.internal_set(record, None).await {
             Ok(_) => Ok(()),
-            Err(error) => Err(StorageError::SetData(error.to_string())),
+            Err(error) => {
+                error!("MySQL error {}", error);
+                Err(StorageError::Other(format!("MySQL Error {}", error)))
```

Comment:
nit: Maybe a `Database` or `Operation` variant? `Other` is fine too though!

Reply:
Yeah `Other` is generally going to be some operation we could retry. If we needed (eventually) an error code which we _should not_ retry, then I'd say at that point to add something. But I think this is OK for now